src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/CompletionHandler.cs

```diff
@@ -181,6 +183,39 @@ internal async Task<(bool, SumType<CompletionItem[], CompletionList>?)> TryGetPr
             return (true, result);
         }
 
+        private SumType<CompletionItem[], CompletionList>? IncludeCSharpKeywords(SumType<CompletionItem[], CompletionList> completionResult, LanguageServerKind kind)
+        {
+            var result = completionResult.Match<SumType<CompletionItem[], CompletionList>?>(
+                items =>
+                {
+                    var newList = items.Union(_keywordCompletionItems, CompletionComparer.Instance);
```

Comment:
If we're concerned about using Linq here I can modify this section to not use Union, but I figured since we were already using Linq I'd avoid premature optimization and just use existing logic.